keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -151,6 +151,25 @@ kpxcForm.onSubmit = function() {
 var kpxcFields = {};
 
 kpxcFields.inputQueryPattern = 'input[type=\'text\'], input[type=\'email\'], input[type=\'password\'], input[type=\'tel\'], input[type=\'number\'], input:not([type])';
+
+// copied from Sizzle.js
+kpxcFields.rcssescape = /([\0-\x1f\x7f]|^-?\d)|^-$|[^\0-\x1f\x7f-\uFFFF\w-]/g;
+kpxcFields.fcssescape = function(ch, asCodePoint) {
+        if (asCodePoint) {
+
+             // U+0000 NULL becomes U+FFFD REPLACEMENT CHARACTER
+            if (ch === "\0") {
+                return "\uFFFD";
```

Comment:
Do not use double quotes.

Reply:
"\0" and "\uFFFD" are not literal strings and should not be written with single quotes

Reply:
These are escaped characters, not literal strings

Reply:
See:

https://github.com/jquery/sizzle/blob/7d92424c1a65a367a45034803e9de97462ea42e5/src/sizzle.js#L154-L171

Reply:
Where was the page you could test this PR? You have removed the description and the original issue.

Reply:
You closed the issue, but it is here:

https://github.com/keepassxreboot/keepassxc-browser/issues/429

Reply:
I tested this with my local test file using single quotes. Everything worked just fine.

Reply:
Yes, but the single quotes worked because you didn't have any null characters in there.

`'\0' !== "\0"`

Reply:
Even that comparison worked for me.

Reply:
Chrome has looser rules when it comes to interpreting escaped string literals. I think it should stay as double quotes for maximum cross-browser/version/OS compatibility.

Reply:
I tested this with Firefox. I suggest you use single quotes so we can maintain the code style.

Reply:
Done